packages/liveblocks-react-ui/src/primitives/Markdown.tsx

```diff
@@ -586,23 +627,18 @@ function InlineTokenComp({
         ));
       }
 
-      const Anchor = components?.Anchor ?? defaultComponents.Anchor;
+      const Link = components?.Link ?? defaultComponents.Link;
```

Comment:
I forgot the difference between "Anchor" and "Link" when I added this last time: "Anchor" is a way to override _any_/_all_ links in the **default components** (e.g. needing to use `next/link` instead of `a`), "Link" was used in the `Comment.Body` primitive to [allow customizing](https://liveblocks.io/docs/api-reference/liveblocks-react-ui#primitives-Comment.Body-Link) `CommentBody` links (e.g. style them, add tooltips, etc), the same exists for links inside `Composer.Editor`. Since the case of `Markdown` is the same as `Comment.Body`/`Composer.Editor`, I'm changing it to "Link".